share/man/man9/buf.9

```diff
@@ -132,8 +132,7 @@ instantiated VM Buffers (struct buf's) prevent their underlying pages in the
 buffer cache from being freed.
 This can complicate the life of the paging
 system.
-.\" .Sh SEE ALSO
-.\" .Xr <fillmein> 9
+
```

Comment:
Likely don't want a blank line here

Reply:
blank line address in subsequent commit.